data/uhc_pack/functions/create_scoreboards.mcfunction

```diff
@@ -18,8 +18,13 @@ scoreboard objectives add uhcLobby dummy "Game Details"
 scoreboard objectives add uhcOpt trigger
 scoreboard objectives add uhcTeams dummy
 
+# Dynamic teleportation scoreboard
+scoreboard objectives add uhcDynTP dummy "Dynamic TP"
+
 # Pre-generation scoreboards
 scoreboard objectives add uhcPG dummy "Pre-generation"
+scoreboard objectives add uhcPGSet dummy "Pre-generation settings"
+scoreboard objectives add uhcPGQueue dummy "Pre-generation unload queue"
```

Comment:
This is fine to keep as a separate scoreboard due to it being armorstand independent